Jenkinsfile

```diff
@@ -22,9 +21,6 @@ pipeline {
           ${WORKSPACE}/env/bin/pip install -r requirements.dev.txt
           # make sure pytest inmanta is the required version
           '''
-          if (params.pytest_inmanta_dev) {
```

Comment:
Remote the comment `# make sure pytest inmanta is the required version` as well

Reply:
good catch, fixed.